build.sbt

```diff
@@ -914,7 +914,8 @@ lazy val httpUtils = (project in engine("httpUtils")).
         "com.typesafe.scala-logging" %% "scala-logging" % scalaLoggingV,
         "com.softwaremill.sttp.client" %% "core" % sttpV,
         "com.softwaremill.sttp.client" %% "async-http-client-backend-future" % sttpV,
-        "com.softwaremill.sttp.client" %% "circe" % sttpV
+        "com.softwaremill.sttp.client" %% "circe" % sttpV,
+        "com.softwaremill.sttp.client" %% "akka-http-backend" % sttpV,
```

Comment:
Why this dependency is in `http-util` instead of `security`? `http-util` is used by model extensions and we don't want to have akka in model ...